scripts/setup

```diff
@@ -25,6 +25,8 @@ DATA_PATH=$(pipenv --where)/data
 #       used for create only the items and holdings files for the 'big' documents file
 #  --deployment:
 #       used for deploy the 'big' files
+#  --extra_param:
```

Comment:
Do we like to give multiple parameter as extra parameter?
Otherwise it should be better to give a better name to the variable.